### DISCUSSION TITLE
React + Typescript support

### DISCUSSION THREAD

**Discussion by doraemonxxx** [vbenjs/vue-vben-admin#1895] [General]
### React JS/TS support

Is there any implementation of vben-admin for React? If not, I would like to suggest a React TS version of the app. That would be nice because many devs are working with react nowadays (the reasons are React js jobs are stable and in-demand), and the current implementation of vben-admin + typescript is robust. Also, ant-design(react version) components and documentation with examples are far way better than ant-design-vue.

If vben-admin react+ts will be implemented, that would be 100% awesome.

**Comment by jpainam** [comment]
The design looks great, anyone working on a react version?

**Reply by doraemonxxx** [reply]
Nope. This project used Vue only as a framework. you have to work on your own if you want to use React
